rpc.go

```diff
@@ -20,25 +20,114 @@
 
 package yarpc
 
-import "github.com/yarpc/yarpc-go/transport"
+import (
+	"fmt"
+
+	"github.com/yarpc/yarpc-go/sync"
+	"github.com/yarpc/yarpc-go/transport"
+
+	"golang.org/x/net/context"
+)
 
 // RPC TODO
 type RPC interface {
+	transport.Handler
 	transport.Registry
 
 	// Retrieves a new Outbound transport that will make requests to the given
 	// service.
+	//
+	// This panics if the given service is unknown.
 	Channel(service string) transport.Outbound
+	// TODO do we really want to panic on unknown services?
```

Comment:
:no_good: Rather not. At some point in the future, perhaps we can default to a stub that returns the YARPC version of 404s for all procedure calls.